lib/api/src/getSize.js

```diff
@@ -10,7 +10,7 @@ export default function (path, cb) {
     } else {
       let fileLength = 0
       for (let i = 0; i < file.EndofFile.length; i++) {
-        fileLength |= file.EndofFile[i] << (i * 8)
+        fileLength += file.EndofFile[i] * Math.pow(2, i * 8);
```

Comment:
We follow https://github.com/feross/standard

Please do not add `;` at line endings :)